### DISCUSSION TITLE
런타임중에 화면 회전이 가능한가요?

### DISCUSSION THREAD

**Discussion by minsuslash** [naverz/zepeto-studio-kor#1283] [5. Q&A]
런타임중에 가로화면 모드에서 세로화면 모드로 화면 회전이 가능한지 궁금합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 유니티로 개발한 월드는 Open World Settings에서 설정한 Orientation 모드대로만 보여지게 됩니다.
개발자들이 설정한 모드에 따른 해상도를 고려해서 UI 및 공간 배치를 하기 때문에 
런타임 중 모드를 변경할 경우 의도대로 동작하지 않을 수 있기 때문입니다. 
감사합니다.
